test_env_setup.sh

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
```

Comment:
It might be a good idea to allow sourcing this script from other directories not only from the current one.
The TEST_CODECHECKER_PACKAGE_DIR might be set to a wrong directory.
See the removed init.sh for example https://github.com/Ericsson/codechecker/commit/d4ab4c94e4fc301b2cbed4e0ab4a0696f36511b8